Doc/library/ssl.rst

```diff
@@ -1789,6 +1817,10 @@ to speed up repeated connections from the same clients.
 
    .. versionadded:: 3.3
 
+   .. deprecated:: 3.15
```

Comment:
This should be done in a separate PR as it would be better to raise a warning. Leave it supported for now.

Reply:
Understood - reverted for now.